core/src/components/chip/chip.tsx

```diff
@@ -36,14 +36,21 @@ export class Chip implements ComponentInterface {
    */
   @Prop() disabled = false;
 
+  /**
+   * Define the Chip corner shape, when using the Ionic Theme.
+   */
+  @Prop({ reflect: true }) shape?: 'soft' | 'round' | 'rectangular';
```

Comment:
Can we create a tech debt ticket to make sure we add these features back to the iOS/MD themes?

Reply:
Sure, the ticket should be created by someone on your team I guess? @sean-perkins

Reply:
I created this ticket for you: https://outsystemsrd.atlassian.net/browse/FW-6120

Reply:
thank you!!